src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Editor/PassingTimesEditor/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from 'react';
```

Comment:
Hele den fila her virker unødvendig komplisert må jeg si 😛 Greit nok at vi ikke trenger å nøste opp dette nå, dersom vi vil ha ut disse endringene relativt fort. Men greit å skrive bak øret at den egentlig kan skrives helt om. Litt unødvendig å skulle ha en `getTimePicker` o.l. når det kun er en TimePicker per rad osv :) Blir sikkert enklere å nøste opp når `withFieldChanges` er borte og

Reply:
Yes, helt enig att det kan skrivas mycket enklare! Men allting blir lite mer komplicert när man ska ta bort 3 av 4 input fields och sørge for att bara den ena input field:en fortsatt kan hantera allt funktionalitet och logik 😊.